src/Frame.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __FRAME_H__
+#define __FRAME_H__
+
+#include <string>
+#include <cstdint>
+
+using namespace std;
```

Comment:
I prefer to not  `using namespace std`; Prefixing stuff in the std namespace with `std::` helps avoid any confusion with namespaces we may define.